rasa_core/interpreter.py

```diff
@@ -24,6 +25,8 @@ def create(obj, endpoint=None):
         from rasa_nlu.model import Interpreter
 
         if (isinstance(obj, NaturalLanguageInterpreter) or
+                # TODO: I doubt we should allow the NLU interpreter here!
```

Comment:
done: reached out to tobias, let's say what he says (cc @wochinge )

Reply:
I use this here https://github.com/RasaHQ/rasa_stack/blob/c8207638e095535abfec5b250995a80b561d81df/rasa/jupyter.py#L23 . Won't this be unified anyway when we merge Core and NLU?

Reply:
We should add a comment to the interpreter then and create an issue (otherwise we'll forget it). Would also be good if you create a PR and add the missing `message_id` parameter to NLU. 

We need to be very careful here because they do not have the sam signature for the parse method, and this PR makes it even worse as the interpreter.parse is now async in core.

Reply:
mhm, or I rewrite the jupyter thing that it does not take the `interpreter` but rather the model path and then use `RasaNLUInterpreter(model_directory=nlu_path)`

Reply:
yes let's do that :+1:

Reply:
you can also be a little more sneaky and do something like
```
core_interpreter = RasaNLUInterpreter(nlu_path, lazy_init=True)
core_interpreter.interpreter = interpreter
```
to avoid loading the interpreter again :)

Reply:
:+1: yes, that's what I've been planning todo :-)

Reply:
ah, that's really sneaky! :-D